shapely/tests/test_ragged_array.py

```diff
@@ -317,8 +318,10 @@ def test_multilinestrings():
 def test_multipolygons():
     arr = shapely.from_wkt(
         [
-            "MULTIPOLYGON (((35 10, 45 45, 15 40, 10 20, 35 10), (20 30, 35 35, 30 20, 20 30)))",
-            "MULTIPOLYGON (((40 40, 20 45, 45 30, 40 40)),((20 35, 10 30, 10 10, 30 5, 45 20, 20 35),(30 20, 20 15, 20 25, 30 20)))",
+            "MULTIPOLYGON (((35 10, 45 45, 15 40, 10 20, 35 10), "
+            "(20 30, 35 35, 30 20, 20 30)))",
+            "MULTIPOLYGON (((40 40, 20 45, 45 30, 40 40)),"
+            "((20 35, 10 30, 10 10, 30 5, 45 20, 20 35),(30 20, 20 15, 20 25, 30 20)))",
             "MULTIPOLYGON EMPTY",
```

Comment:
For cases like this a `  # noqa: E501` might be nicer?

Reply:
Same to me... I'll change it...